Fix makefile

weineran opened the pull request:
* When running `make install`, I got this error:

      Installing to: invalid value "./etrade" for flag -C: -C flag must be first flag on command line
      usage: go list [-f format] [-json] [-m] [list flags] [build flags] [packages]
      Run 'go help list' for details.
      make: *** [Makefile:11: install] Error 2

* I ran 'go help list' but the -C flag was undocumented.
* I tried moving the -C flag to the first position after "go list", but I got this error:

      Installing to: go: chdir -f: no such file or directory
      make: *** [Makefile:11: install] Error 1

* I ultimately resolved the error by removing the -C flag. `make install` now runs successfully.

weineran commented:
Howdy @jerryryle ! Thanks for making this tool. I can't believe it doesn't have more stars and forks!

I'm using your tool to wrap my head around the eTrade API because, as you wrote in your README, the API is poorly documented.

In order to get your tool installed, I had to make a tweak to the makefile, so figured I'd open a PR. (Although it could have just been user error by me...)

jerryryle commented:
Thanks for the PR! I'll review asap. I can't remember why I needed the -C flag in the first place, so let me figure that out & confirm whether your suggestion is the best course of action.

jerryryle commented:
Are you able to confirm what version of Go you have installed (`go version`)? It looks like the `-C` flag [was added in v1.20](https://go.dev/doc/go1.20), so if you have a version older than that, it might explain your error.

weineran commented:
I am running 1.21.5 (on Windows Subsystem for Linux)
```
go version
go version go1.21.5 linux/amd64
```

weineran commented:
Found some info:
https://tip.golang.org/doc/go1.21
```
The -C dir flag must now be the first flag on the command-line when used.
```
But I tried playing around with the order of flags, and still couldn't get it to work. But again, could have been user error.

weineran commented:
The original issue was, because I couldn't find any clear docs on the `-C` option, I was treating it as a flag. i.e. I just moved `-C` to the first position, which of course didn't work. I now understand it was an _option_. So I have moved `-C ./etrade` to the first position. That works now. (Using go version 1.21)

You might want to confirm if it still works in version 1.20, but I think it will.

jerryryle commented:
I reproduced the issue and confirmed that this fixes it. Thank you!

weineran commented:
Great!

On Fri, Jan 12, 2024 at 9:37 AM Jerry Ryle ***@***.***> wrote:

> I reproduced the issue and confirmed that this fixes it. Thank you!
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/jerryryle/etrade-cli/pull/10#issuecomment-1889617029>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ABNAEIID2KASXGAZPDMSHRLYOFRC3AVCNFSM6AAAAABBSGVJQKVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMYTQOBZGYYTOMBSHE>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>
